components/Astronaout.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import Lottie from 'react-lottie';
+import animationData from '../ lotties/astronaut.json';
+
+class AstronautLottie extends Component {
+  render() {
```

Comment:
Rather than using a class here we should use the functional version:
```
export default function AstronautLottie()
```
As a part of this we can remove the render() and just use the return as shown with other components.

Reply:
@ozer619 Hey, let us know if you need any help with figuring out this change. 😃